ParkingLot/ICar.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ParkingLot
+{
+    public interface ICar
```

Comment:
good design of interface, but overdesign?

Reply:
For car, I agree.